src/java/fr/paris/lutece/plugins/mylutece/modules/cacheuserattribute/business/CacheUserAttributeDAO.java

```diff
@@ -226,4 +228,34 @@ public Optional<CacheUserAttribute> loadByUserAndAttributeId( String idUser, int
         }
     }
 
+    /**
+     * {@inheritDoc }
+     */
+    @Override
+    public List<CacheUserAttribute> selectByListUserIdsAndAttributeId( List<String> listUserIds, int nAttributeId, Plugin plugin )
+    {        
+        try ( DAOUtil daoUtil = new DAOUtil( SQL_SELECT_BY_LIST_USER_ID + " '" + String.join( "','", listUserIds ) + "' ) " + " AND " + SQL_FILTER_BY_ATTR_ID, plugin ) )
```

Comment:
There is a small risk of SQL injection, it would be better to make a loop to prepare the statement with "?", and then a loop to add the attributes, which will also handle the case where the ids contain quotes (we don't know in advance what the user ids will look like)